src/main/java/ch/njol/skript/expressions/ExprSecCreateWorldBorder.java

```diff
@@ -22,16 +22,28 @@
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
 
-@Name("Create WorldBorder")
-@Description("Create a virtual worldborder. You can make this a real border by setting a world's worldborder to a virtual border.")
+@Name("Create Virtual WorldBorder")
+@Description({
+    "Creates a new virtual worldborder that can be customized inside a section.",
+    "Each time this expression is used, a fresh worldborder is created.",
+    "You can apply the customized worldborder to a player or to a world.",
+    "Setting a world's worldborder to a virtual border makes it a real, visible border."
+})
 @Examples({
-	"set player's worldborder to a virtual worldborder",
-	"",
-	"on join:",
-		"\tset {_border} to a worldborder:",
-			"\t\tset worldborder radius to 25",
-			"\t\tset worldborder warning distance of event-worldborder to 5",
-		"\tset worldborder of player to {_border}"
+    "on join:",
+    "\tset player's worldborder to a virtual worldborder",
+    "\tset {_center} to location of player",
+    "\tset {_border} to a virtual worldborder:",
+    "\t\tset worldborder radius to 50",
+    "\t\tset world border center of event-worldborder {_center}",
+    "\tset player's worldborder to {_border}",
+    "",
+    "on join:",
+    "\tset {_worldborder} to a virtual worldborder:",
+    "\t\tset worldborder radius to 200",
+    "\t\tset worldborder center of event-worldborder to location(0, 64, 0)",
+    "\t\tset worldborder warning distance of event-worldborder to 5",
+    "\tset worldborder of world \"world\" to {_worldborder}"
 })
```

Comment:
Since we're making multiple examples this might benefit from using the `@Example` annotation which is repeatable
```java
@Example{"""
      on join:
          blank1
"""}
@Example{"""
      on quit:
          blank1
"""}
```

If possible can we use a multiple events you can keep `on join` but maybe try one with a function that passes a player parameter?

Reply:
fyi those multiline strings are improperly formatted, the ending """ need to be on the same column as on quit/on join.

Reply:
¯\_(ツ)_/¯ I forget how it goes, but also hate how ya'll format things, Java will remove blanks and indentions to the lowest  amount 

I find that way more readable at a glance

Reply:
It's not to do with any readability concerns lol
The ending """ determines where the quote's indentation starts. The formatting you have makes it so on quit is indented by 2 tabs in the string, rather than 0.